style.css

```diff
@@ -23,28 +23,28 @@
   background-color: #616161;
 }
 
-.burger:before {
+.burger::before {
   top: -8px;
 }
 
-.burger:after {
+.burger::after {
   bottom: -8px;
 }
 
-@media (max-width:400px) {
-.content__description { font-size: 3px; } 
-   }
-    @media (min-width:401px) and (max-width:599px) { .content__description { font-size: 6px; }
-}
-@media (min-width:600px) and (max-width:767px) { 
-.content__description { font-size: 9px; } 
-}
-@media (min-width:768px) and (max-width:950px) { 
-.content__description { font-size: 12px; }
+@media (max-width:700px) {
+  .content__description { font-size: 13px; }
+   .menu__link {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/HelenRom3/pullRequest?prid=6002362)